src/dataframe/dataframe.jl

```diff
@@ -1128,8 +1128,11 @@ end
 
 """
     append!(df1::DataFrame, df2::AbstractDataFrame; cols::Symbol=:setequal)
+    append!(df::DataFrame, x; cols::Symbol=:setequal)
```

Comment:
Maybe call `x` something more explicit, like `rows`?

Reply:
OK. I called it `table`